app/assets/modules/Import/Tasks/Import.tsx

```diff
@@ -194,7 +218,12 @@ class Import extends React.Component<IProps, IState> {
           </Button>
           <Button
             className="import-again"
-            onClick={() => this.props.stopImport({ taskId })}
+            onClick={() =>
+              this.props.stopImport({
+                taskID: taskId,
```

Comment:
can keep the same style? taskID => taskId?

Reply:
> can keep the same style? taskID => taskId?
its defined in gateway